cartographer_ros_msgs/msg/SubmapEntry.msg

```diff
@@ -12,5 +12,5 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-uint32 submap_version
+int32 submap_version
```

Comment:
Thanks. Since we always squash and merge, submitting it as separate PRs allows us to easily rollback if necessary.